_non-cosmos/ton/assetlist.json

```diff
@@ -39,6 +37,42 @@
         "website": "https://ton.tg/",
         "twitter": "https://x.com/ton_blockchain"
       }
+    },
+    {
+      "description": "The Hamster Kombat ($HMSTR) is the cryptocurrency of the TON (The Open Network) blockchain.",
+      "extended_description": "HMSTR (The Hamster Kombat) is a cryptocurrency-based clicker game that has garnered attention for its engaging gameplay and integration with the digital currency ecosystem.",
+      "denom_units": [
+        {
+          "denom": "nanohmstr",
+          "exponent": 0,
+          "aliases": ["nanoHmstr"]
+        },
+        {
+          "denom": "hmstr",
+          "exponent": 9
+        }
+      ],
+      "type_asset": "unknown",
+      "base": "nanohmstr",
+      "name": "Hamster",
+      "display": "hmstr",
+      "symbol": "HMSTR",
+      "coingecko_id": "hamster-kombat",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/hmstr.png",
```

Comment:
Please relocate the uploaded images to the _non-cosmos/ton/images/ directory (because that's where the asset originates).